lib/matplotlib/axes/_axes.py

```diff
@@ -7194,9 +7194,16 @@ def stairs(self, values, edges=None, *,
             True or an array is passed to *baseline*, a closed
             path is drawn.
 
+            If None, then drawn as an unclosed Path.
+
         fill : bool, default: False
             Whether the area under the step curve should be filled.
 
+            Passing both ``fill=True` and ``baseline=None`` will likely result in
+            undesired filling as the first and last points will be connected
+            with a straight line with the fill between the line and the stairs.
```

Comment:
```suggestion
            undesired filling: the first and last points will be connected
            with a straight line and the fill will be between this line and the stairs.
```
I found this sentence a little hard to parse.